src/styles/global/normalize.scss

```diff
@@ -1,163 +1,171 @@
-/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v6.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Yeah, seems to just be some small tweaks: https://github.com/necolas/normalize.css/commit/b4a8fdaf8321093b1e3cad612fb76cf2b7d3275d

Should be fine to upgrade to v7.0.0.